pkg/couchdb/database/.eslintrc.cjs

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+	root: true,
+	parser: '@typescript-eslint/parser',
+	extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended', 'prettier'],
+	plugins: ['svelte3', '@typescript-eslint'],
+	ignorePatterns: ['*.cjs'],
+	overrides: [{ files: ['*.svelte'], processor: 'svelte3/svelte3' }],
+	settings: {
+		'svelte3/typescript': () => require('typescript')
+	},
+	parserOptions: {
+		sourceType: 'module',
+		ecmaVersion: 2020
+	},
+	env: {
+		browser: true,
+		es2017: true,
+		node: true
+	}
```

Comment:
Ok...now you want to use typescript parser (requiring appropriate tsconfig). Since we're using the default `<root>/tsonfig.json`, this should be loaded automatically, but sometimes, it just doesn't work. 
There's a helper for that in `pkg/scaffold/.eslint.utils.js` (`useTSConfig`). You need to apply it here, you can use the `pkg/shared` as a reference (the config should stay as is, only be wrapped with the helper).